### ISSUE TITLE
`ng2 opt` exception

### ISSUE THREAD

**Issue by blai** [SamVerschueren/alfred-ng2#3]
I tried with the most basic example shown in README, but encounter the following exception:

```
TypeError: TYPES.includes is not a function
    at /Users/brianlai/Library/Application Support/Alfred 3/Alfred.alfredpreferences/workflows/user.workflow.1EE06DB4-AB90-48CB-88EC-F8B876DDF25F/index.js:32:17
    at /Users/brianlai/Library/Application Support/Alfred 3/Alfred.alfredpreferences/workflows/user.workflow.1EE06DB4-AB90-48CB-88EC-F8B876DDF25F/index.js:23:25
```

**Comment by SamVerschueren** [comment]
Update Node.js to 6+. Might remove `.includes` though and use the good old `.indexOf` to support Node.js 4 as well. What version are you using?

**Comment by blai** [comment]
I see, I constantly switch between node 6 and node 4 (for development sake), when I encounter this, it was Node 4.4.3

**Comment by SamVerschueren** [comment]
Fixed using `.indexOf` instead of `.includes`. Just released `v0.3.2`, so update the workflow and you should be good to go.
